great_expectations/expectations/core/expect_column_quantile_values_to_be_between.py

```diff
@@ -147,29 +147,28 @@ class ExpectColumnQuantileValuesToBeBetween(ColumnExpectation):
         "profiler_config",
     )
 
-    quantile_value_ranges_estimator_parameter_builder_config: ParameterBuilderConfig = (
-        ParameterBuilderConfig(
-            module_name="great_expectations.rule_based_profiler.parameter_builder",
-            class_name="NumericMetricRangeMultiBatchParameterBuilder",
-            name="quantile_value_ranges_estimator",
-            metric_name="column.quantile_values",
-            metric_domain_kwargs=DOMAIN_KWARGS_PARAMETER_FULLY_QUALIFIED_NAME,
-            metric_value_kwargs={
-                "quantiles": f"{VARIABLES_KEY}quantiles",
-                "allow_relative_error": f"{VARIABLES_KEY}allow_relative_error",
-            },
-            enforce_numeric_metric=True,
-            replace_nan_with_zero=True,
-            reduce_scalar_metric=True,
-            false_positive_rate=f"{VARIABLES_KEY}false_positive_rate",
-            estimator=f"{VARIABLES_KEY}estimator",
-            num_bootstrap_samples=f"{VARIABLES_KEY}num_bootstrap_samples",
-            bootstrap_random_seed=f"{VARIABLES_KEY}bootstrap_random_seed",
-            truncate_values=f"{VARIABLES_KEY}truncate_values",
-            round_decimals=f"{VARIABLES_KEY}round_decimals",
-            evaluation_parameter_builder_configs=None,
-            json_serialize=True,
-        )
+    quantile_value_ranges_estimator_parameter_builder_config: ParameterBuilderConfig = ParameterBuilderConfig(
+        module_name="great_expectations.rule_based_profiler.parameter_builder",
+        class_name="NumericMetricRangeMultiBatchParameterBuilder",
+        name="quantile_value_ranges_estimator",
+        metric_name="column.quantile_values",
+        metric_domain_kwargs=DOMAIN_KWARGS_PARAMETER_FULLY_QUALIFIED_NAME,
+        metric_value_kwargs={
+            "quantiles": f"{VARIABLES_KEY}quantiles",
+            "allow_relative_error": f"{VARIABLES_KEY}allow_relative_error",
+        },
+        enforce_numeric_metric=True,
+        replace_nan_with_zero=True,
+        reduce_scalar_metric=True,
+        false_positive_rate=f"{VARIABLES_KEY}false_positive_rate",
+        quantilie_statistic_interpolation_method=f"{VARIABLES_KEY}quantilie_statistic_interpolation_method",
```

Comment:
I believe this is a typo. Should be `quantile_statistic_interpolation_method`

Reply:
Fixed -- great catch, thanks @Shinnnyshinshin !